test/framework/fixtures/libgoalFixture.go

```diff
@@ -499,3 +502,50 @@ func (f *LibGoalFixture) MinFeeAndBalance(round uint64) (minFee, minBalance uint
 	}
 	return params.MinTxnFee, minBalance, nil
 }
+
+// ProofRespToProof returns a proof for usage in merkle array verification from the inputted API proof query response.
+func (f *LibGoalFixture) ProofRespToProof(proofResp generatedV2.ProofResponse) (proof merklearray.Proof, err error) {
```

Comment:
It's possible, and have it return both the response and the proof itself? That's why I didn't have it this way, to begin with - it will have to return both values.

Reply:
Done, take a look and let me know what you think